public/locales/bg/common.json

```diff
@@ -1,81 +1,82 @@
 {
```

Comment:
Please undo the whitespace change of this file, seems unecessary

Reply:
Ready.